### DISCUSSION TITLE
Can add Onedrive Business account?

### DISCUSSION THREAD

**Discussion by earthmoon** [bpozdena/OneDriveGUI#76] [Q&A] [answered]
Hello,
When adding Onedrive Business account, I click the click the link and input my Username and Password to login, which leads me to a white page. Can Onedrive Business account be added?
Thanks.

**Comment by bpozdena** [comment, marked as answer]
Yes, business account even with external SAML authentication should work just fine. 

If you are using the AppImage version, whitepage is probably expected in your browser after you successfully login. You just need to copy to URL of the blank page into the OneDriveGUI wizard. 

If this is not clear, please specify if you are using the AppImage version and/or provide a screenshot (you can blur sensitive details, domains, etc.).

**Reply by earthmoon** [reply]
Thanks for your kindle reply. I do use the AppImage, and after I copy that link to save, it works. But may you teach me why AppImage will cause whitepage? Thanks.
